cmd/root.go

```diff
@@ -264,3 +269,22 @@ func Execute() {
 		os.Exit(1)
 	}
 }
+
+func checkForPremiumError(app *view.App, jobsController *controller.Jobs) {
+	for {
+		runningJobs := jobsController.FailedJobs()
```

Comment:
this for loop increases cpu usage. add time.sleep after each loop. 500 ms should be enough